src/main/java/no/nav/fo/veilarbregistrering/oppfolging/AktiverBrukerFeil.java

```diff
@@ -0,0 +1,24 @@
+package no.nav.fo.veilarbregistrering.oppfolging;
+
+import java.util.Optional;
+
+import static java.util.Optional.ofNullable;
+
+enum ArenaFeilType {
+    BRUKER_ER_UKJENT,
+    BRUKER_KAN_IKKE_REAKTIVERES,
+    BRUKER_ER_DOD_UTVANDRET_ELLER_FORSVUNNET,
+    BRUKER_MANGLER_ARBEIDSTILLATELSE
+}
+
+public class AktiverBrukerFeil {
+    private String id;
+    private ArenaFeilType type;
+    private String detaljer;
```

Comment:
Hvis vi ikke skal bruke de 2 andre feltene, så kan vi droppe å ha de med. Da er vi mindre sårbare for endringer i kontrakten.